app/controllers/api/abstract_controller.rb

```diff
@@ -19,6 +19,18 @@ class AbstractController < ApplicationController
 
 private
 
+    def authenticate_user!
+      return true if current_user
+      auth = Auth::Create.run(request.headers.to_h)
+      if auth.success?
+        @current_user = auth.result
+      else
+        sorry("""You failed to authenticate with the API. Ensure that you have
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.